index.js

```diff
@@ -19,24 +29,43 @@ const cacheReply = function(fn, cacheOptions = {}) {
     // return cache stats if requested:
     if (query[statsQueryParam]) {
       return {
+        statusCode: 200,
         headers: {
           'content-type': 'application/json; charset=utf8'
         },
-        body: JSON.stringify(cache.cache.getStats())
+        body: JSON.stringify(cache.getStats())
       };
     }
-    // skip cache altogether if requested:
-    if (query[skipQueryParam]) {
+    // skip cache altogether if requested or explicitly disabled:
+    if (query[skipQueryParam] || cacheOptions.enabled === false) {
+      if (cacheOptions.log) {
```

Comment:
what about having log default to a function that does nothing?  might be cleaner than checking everywhere